src/BookReader.tsx

```diff
@@ -2,23 +2,26 @@ import { Asset } from 'expo-asset'
 import * as FileSystem from "expo-file-system";
 import * as React from "react";
 import { FunctionComponent } from 'react';
-import { Platform, SafeAreaView, StyleSheet, } from 'react-native';
+import { Dimensions, Platform, StyleSheet } from 'react-native';
 import { WebView } from 'react-native-webview';
 
 import Constants from 'expo-constants';
 
-type BookReaderProps = {
+interface BookReaderProps  {
     bookUrl: string
 }
 
+const {height, width} = Dimensions.get('window');
+
 export const BookReader: FunctionComponent<BookReaderProps> = (props) => {
     console.log('in BookReader');
     // react-native-webview has a bug on Android where local URI sources print out the HTML as text instead of as HTML
     // (See https://github.com/react-native-webview/react-native-webview/issues/428 and https://github.com/react-native-webview/react-native-webview/issues/518)
     // For now, we'll just have Android reference the online version
-    const baseUri = Platform.OS === "android"
+    let baseUri = Platform.OS === "android"
```

Comment:
We can go back to const

Reply:
Done. Also deleted a color in the ReaderScreen we weren't using anymore.